packages/scoutgame/src/points/builderEvents/sendPointsForMiscEvent.ts

```diff
@@ -25,6 +25,20 @@ export async function sendPointsForMiscEvent({
   tx?: Prisma.TransactionClient;
 }) {
   const season = getCurrentSeason(week).start;
+  // make sure the sender has enough points
+  if (points < 0) {
```

Comment:
added support to send negative points to take them away from a user. this way we don't need a new special builder event type